mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -74,12 +74,26 @@ def stop_app(self):
 
     def _is_app_installed(self):
         """Overrides superclass."""
+        # Check if app is installed
+        try:
```

Comment:
I thought about that, but it seemed like coming up with a proper design and usage for this 'wrapper' would be more complicated than just writing it out.

Since we don't know what will go into the try block, we'd have a function that takes a function?
```
def adb_stuff():
  out = self.adb.blah()
  # do sth with out.

wrapper(adb_stuff)
```

or if you're thinking about a one line wrapper that returns the `out`, I was actually thinking about adding a param in Adb proxy itself for that.

Either way, it seems like a bigger change that deserves its own CL?